scripts/publish.ts

```diff
@@ -83,12 +83,7 @@ async function main(dryRun = false) {
 
   await run(
     'packages/fetch-engine',
-    `pnpm i @prisma/engines-version@${newVersion}`,
-    dryRun,
-  )
-  await run(
-    'packages/fetch-engine',
-    `pnpm i @prisma/get-platform@${newVersion}`,
+    `pnpm i @prisma/engines-version@${newVersion} @prisma/get-platform@${newVersion}`,
```

Comment:
What is the benefit of doing it like this vs. how it was?

Reply:
Basically, because the local package versions have already been adjusted so if you try to update the local packages individually you then run in to an issue where pnpm can not resolve one of the packages. 

For example, pnpm would try to update `@prisma/engines-version` to (workspace:2.27.0-23.hash) but would fail as `@prisma/get-platform` would be pinned to (workspace:2.27.0-22.hash) but the local package has already been updated to `23` so it can't be resolved.

Reply:
I would by lying if I said I understood this, but it sounds smart so I will accept this as a great answer :D